rbac/management/management/commands/utils.py

```diff
@@ -178,19 +178,25 @@ def batch_import_workspace(records):
         for record in records:
             is_ungrouped = record["ungrouped"].lower() == "true"
             parent = parent_workspace_dict[record["org_id"]]
-            workspace_type = Workspace.Types.UNGROUPED_HOSTS if is_ungrouped else Workspace.Types.STANDARD
+            if is_ungrouped:
+                ws_name = "Ungrouped Hosts"
```

Comment:
Maybe we can fix the "Ungrouped Hosts" workspace name in a variable?

Reply:
Good idea